packages/twenty-server/src/engine/core-modules/workspace/services/workspace.service.ts

```diff
@@ -49,7 +49,7 @@ export class WorkspaceService extends TypeOrmQueryService<Workspace> {
     return await this.workspaceManagerService.doesDataSourceExist(id);
   }
 
-  async solfDeleteWorkspace(id: string) {
+  async softDeleteWorkspace(id: string) {
```

Comment:
Is it really a softDelete? It seems it's erasing everything 🤔

Reply:
Indeed I don't know why it is called soft delete